requirements/readthedocs.txt

```diff
@@ -0,0 +1,4 @@
+mmengine
```

Comment:
there is a requirements.txt outside the requirements directory. Shall we move it inside?